Resources/public/js/views/services/ez-roleserversideviewservice.js

```diff
@@ -295,5 +299,19 @@ YUI.add('ez-roleserversideviewservice', function (Y) {
                 }
             });
         },
+
+        /**
+         * Refreshes the role view, showing the assignments tab.
+         *
+         * @method _refreshAssignmentsTab
+         * @protected
+         * @param {Object} data
+         */
+        _refreshAssignmentsTab: function (data) {
+            this.get('app').set('loading', true);
+            this.load(Y.bind(function () {
+                this.get('app').navigateTo('adminRole', {uri: 'pjax/role/view/' + data.roleId + '/ez-tabs-content'});
```

Comment:
following https://github.com/ezsystems/PlatformUIBundle/pull/438/files#r46158645, `data.roleId` will be useless so the `roleRestId` renaming becomes useless.

Also for the record, this code does not mean anything. Basically the server side view will do an AJAX request and when it's done you ask the application to navigate somewhere that will again do an AJAX request (almost the same...).